modules/electron/system-part2/module_system_part2_export.ld

```diff
@@ -1,20 +1,20 @@
 link_dynamic_reset_handler_location = platform_system_part2_vector_table_flash_start + 4;
 
 /* Dynalib table locations */
-link_dynalib_location_hal                  = platform_system_part2_dynalib_table_flash_start + 0;
-link_dynalib_location_rt                   = platform_system_part2_dynalib_table_flash_start + 4;
-link_dynalib_location_system               = platform_system_part2_dynalib_table_flash_start + 8;
-link_dynalib_location_hal_peripherals      = platform_system_part2_dynalib_table_flash_start + 12;
-link_dynalib_location_hal_i2c              = platform_system_part2_dynalib_table_flash_start + 16;
-link_dynalib_location_hal_gpio             = platform_system_part2_dynalib_table_flash_start + 20;
-link_dynalib_location_hal_spi              = platform_system_part2_dynalib_table_flash_start + 24;
-link_dynalib_location_hal_core             = platform_system_part2_dynalib_table_flash_start + 28;
-link_dynalib_location_hal_socket           = platform_system_part2_dynalib_table_flash_start + 32;
-link_dynalib_location_hal_cellular         = platform_system_part2_dynalib_table_flash_start + 36;
-link_dynalib_location_hal_usart            = platform_system_part2_dynalib_table_flash_start + 40;
-link_dynalib_location_system_net           = platform_system_part2_dynalib_table_flash_start + 44;
-link_dynalib_location_system_cloud         = platform_system_part2_dynalib_table_flash_start + 48;
-link_dynalib_location_hal_concurrent       = platform_system_part2_dynalib_table_flash_start + 52;
-link_dynalib_location_hal_can              = platform_system_part2_dynalib_table_flash_start + 56;
-link_dynalib_location_hal_rgbled           = platform_system_part2_dynalib_table_flash_start + 60;
-link_dynalib_location_hal_dct              = platform_system_part2_dynalib_table_flash_start + 64;
+PROVIDE (link_dynalib_location_hal                  = platform_system_part2_dynalib_table_flash_start + 0);
```

Comment:
It's weird that if these symbols are not bracketed with `PROVIDE`, the Electron cannot jump to the `main()` function. This can only be produced on Electron so far.